e2e/tests/hmctsSuperUserRemovesOrders_test.js

```diff
@@ -112,14 +113,31 @@ Scenario('HMCTS super user removes a draft order from a case', async ({I, caseVi
   caseViewPage.checkTabIsNotPresent(caseViewPage.tabs.draftOrders);
 });
 
+Scenario('HMCTS super user removes an application from the case', async ({I, caseViewPage, removeOrderEventPage}) => {
+  const newCaseId = await I.submitNewCaseWithData(caseDataWithApplication);
+  await I.navigateToCaseDetailsAs(config.hmctsSuperUser, newCaseId);
+
+  const applicationToRemove = caseDataWithApplication.caseData.additionalApplicationsBundle[0].value;
+  const labelToSelect = 'C2, ' + applicationToRemove.uploadedDateTime;
+
+  await caseViewPage.goToNewActions(config.superUserActions.removeOrdersAndApplications);
+  await removeOrderEventPage.selectApplicationToRemove(labelToSelect);
+  await I.goToNextPage();
+
+  await removeOrderEventPage.addRemoveApplicationReason('Mistakes were made');
+  await I.completeEvent('Submit');
+
+  caseViewPage.checkTabIsNotPresent(caseViewPage.tabs.otherApplications);
```

Comment:
lets store the removed applications on the case, only available to super user? maybe ask business